### PR TITLE
Display daily and monthly yandex limits

### PR DESCRIPTION
**Pull request by rohankatyal29**
Progress bar to show the daily and monthly Yandex API limits
The progress bar appears once 25% of the API limit is reached to prevent UI congestion 
After 100% of the API limit is reached a warning message is shown  

screen shot 2016-07-26 at 10 01 50 am

### CODE REVIEW

**Comment by ankit-m** [comment]
@rohankatyal29 @Ceilican isn't this too much information on the first page. Shouldn't we shift this to **Advanced** tab?

**Comment by rohankatyal29** [comment]
I don't think people who "actually" need it even end up going to the advanced tab

**Comment by rohankatyal29** [comment]
The bar and warning are only displayed when certain conditions are met, they are not always there

**Comment by Ceilican** [comment]
I agree with @rohankatyal29 . This has to be in the basics option, because users might not think of looking at the advanced tab. Nevertheless, perhaps @ankit-m has a point in that this information is being over-emphasized there. The idea of showing the progress bars only after 25% is good, as it reduces the over-emphasis problem. Maybe we could still think of other aesthetic improvements. For instance, we could put the two bars side-by-side, instead of one on top of the other. But anyway, we shouldn't over-think this.

Another suggestion: the bars could be additionally shown in the statistics page too (even when they are less than 25%).

File: lib/scripts/controllers/options.js [hunk 1]
```diff
@@ -114,6 +120,8 @@ export class OptionCtrl {
       this.translatedWordsExist = Object.keys(this.randomTranslatedWordsForQuiz).length === 0 ? false : true;
       this.quizOptions = this.getShuffledValues(this.randomTranslatedWordsForQuiz);
       this.date = new Date();
+      this. yandexMonthlyLimit = this.getYandexLimit('monthly');
```
Comment (Ceilican) [inline lib/scripts/controllers/options.js]: spurious space here?

Comment (rohankatyal29) [inline lib/scripts/controllers/options.js]: Fixed in the lastest commit

**Pull request merged**